### ISSUE TITLE
Add notification that permissions have been assigned

### ISSUE THREAD

**Issue by drzackyll** [unicef/sir-releases#354]
FE will send the following info from a form to the admin:
-name
-unicef email (could be an input for their username with a hardcoded `@unicef.org` after it)
-job title
-duty station

The admin will assign permissions and, once that happens, the user should be informed that they have access.

**Comment by drzackyll** [comment]
relates to #353

**Comment by reinbach** [comment]
@drzackyll We don't currently have `duty_station` in SIR, we have just been using `Region`, `Country`. Do we want to use that info instead of `duty_station`?

**Comment by daniel-tabarcea** [comment]
We had a field called `duty station city` in etools at one point, i'm guessing it's also a city here so plain text if it's blocking you

**Comment by reinbach** [comment]
I guess my question should be more along the lines of why ask for this information for a new user, when we don't really use that info (duty station) in the system. Why not ask for region/country instead.

**Comment by drzackyll** [comment]
Sure, region and country are fine. Since we have to have the admin assign permissions manually anyway, I guess I was just looking for a simple piece of info they could use for that purpose.

**Comment by reinbach** [comment]
endpoint is available at `/api/register` and expects the following payload

```
{
    "name": <string>,
    "email": <email address>,
    "job_title": <string>,
    "region": <region_id>,
    "country": <country_id>,   (optional)
}
```

**Comment by daniel-tabarcea** [comment]
Is this going be the country and region the user is based in, or is it the country and region the user is requesting access for? I'm asking because the way we label the fields should make this clear, if i leave just "country" and "region" it might be confusing to some

**Comment by reinbach** [comment]
Requesting access for.

But you raise an issue with the backend I think, as I'm saving this as the region/country that they are based in as well. I was under the assumption the user would be in the region that they were wanting to "work" in. But I guess that that is not always the case?

**Comment by daniel-tabarcea** [comment]
I have another issue, namely the reliance on static data. We're showing this before the user has access, and before we can call the endpoints to fetch countries and regions. Can we have those in plain text or something similar?

**Comment by reinbach** [comment]
Do we want to make those endpoints (region, country) open?

**Comment by daniel-tabarcea** [comment]
Yes, it's definitly an option. Can we wait for Zack to pitch in on this? I don't know what would be the preferred way forward from a business point of view

**Comment by drzackyll** [comment]
So you're suggesting we open the country/region endpoints so that we can have those as dropdown menus on the landing/request access page?

**Comment by reinbach** [comment]
Yes

**Comment by drzackyll** [comment]
I don't see why not. If we're going to do that, would it make sense to have a dropdown for the actual role? That is, Country Field Security Adviser, Regional Security Adviser, etc)

**Comment by daniel-tabarcea** [comment]
Has this been merged to develop? I'm getting a 404 error when accessing `/api/register/`

**Comment by reinbach** [comment]
Looks like the trailing '/' was causing an issue, I've corrected that.

**Comment by drzackyll** [comment]
@reinbach @daniel-tabarcea on my local, when i click to request, a pop-up prompts me to log in. i imagine this is because the window is attempting to access the list of regions. once i click cancel, permission errors appear in the console. see attachment.

**Comment by drzackyll** [comment]
![Screen Shot 2019-03-05 at 10.27.54 AM.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59b8008c1b0c3c0012343c85/125516c66e82c728ace21e0d46db978826878de9bce1b057e40fa0c5700b77206454f66b7191a379ef0c2956415a4abc30254d58eeb9cb74fce36f34985619e883655922cf3919adbd871eb608ba208a1d8db1bcde4a94fa2137a5dad11b61435c52dcbd4f3fb29843c259f34ee04555a4133e62e474346e0a.png)
